app/models/service.rb

```diff
@@ -362,8 +362,8 @@ def chargeback_report_name
 
   def generate_chargeback_report(options = {})
     _log.info "Generation of chargeback report for service #{name} started..."
-    MiqReportResult.where(:name => chargeback_report_name).destroy_all
-    report = MiqReport.new(chargeback_yaml)
+    MiqReport.where(:name => chargeback_report_name).destroy_all
+    report = MiqReport.create(chargeback_yaml)
     report.queue_generate_table(options)
```

Comment:
@gtanzillo changes made to allow creating MiqReportResult without creating MiqReport, could  you review